docs/UserGuide.md

```diff
@@ -389,25 +389,98 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
+:exclamation: **Take note** The order of how you type the prefixes does not affect the outcome! If you type `sort s/asc n/asc`, Trackermon will still sort by name in ascending order then by status in ascending order
+</div>
+
+Therefore, let's say you want to sort by status in ascending order then by name in ascending order,
+use `sort n/asc s/asc so/statusname`.
+
+<div markdown="block" class="alert alert-warning">
+:exclamation: **Take note** We use `so/SEQUENCE` to swap the priority of the sorting criteria! By default, Trackermon always prioritises sorting in the following order if the criteria are used:
+
+  1. name
+  2. status
+  3. rating
+  4. tag
+
+So, use the "tagratingstatusname" as the `SEQUENCE` if you want to swap the ordering to:
+1. tag
+2. rating
+3. status
+4. name
+
+Additionally, when using `so/` prefix, Trackermon will check if the required criteria is presented in your `SEQUENCE`.
+So if you use `sort n/asc s/asc so/haha`, the required criteria are name and status. As name and status are not spelt in full for `SEQUENCE`, Trackermon will tell you the input is invalid!
+</div>
+
 <div markdown="block" class="alert alert-warning">
 
-:bulb: **Tip:** `Sort` will sort according to the [prefixes](#command-structure).<br>
+:bulb: **Tip:** Summary:
+* `Sort` will sort according to the [prefixes](#command-structure).<br>
 * Use `n/` if you want to sort by name.
-* Use `s/` if you want to sort by status.
-* Use `t/` if you want to sort by tag.
+* Use `s/` if you want to sort by status. We sort status in this ascending order: `completed`, `watching`, `plan-to-watch`.
+* Use `t/` if you want to sort by number of tags.
 * Use `r/` if you want to sort by rating.
 * For the above prefixes, you have to specify the `ORDER` right after the prefix.
 * `ORDER` is the order you want to sort the list by! Type `asc` to sort by ascending or type `dsc` to sort by descending!
-* If two or more of the above prefixes are being used, sort will prioritise sorting by name, then status, followed by rating, and finally tags.
+* The criteria are name, status, tag, rating
+* If two or more of the above criteria are being used, sort will prioritise sorting by name, then status, followed by rating, and finally tags.
 * Use `so/` if you want to reorder the priority and state the `SEQUENCE`
-* List the `SEQUENCE` by listing out the full name of the prefix used in the order you want.
-* For example: you want to sort by name, then status, followed by rating, and finally tags, `SEQUENCE` will be "name status rating tag"
-* `SEQUENCE` input is case-insensitive.
+* List the `SEQUENCE` by listing out the full name of the criteria used in the order you want.
+* For example: you want to sort by rating, then status, followed by name, and finally tag, `SEQUENCE` will be "ratingstatusnametag"
+* `SEQUENCE` input is case-insensitive and space inbetween the criteria does not matter. So the input "RaTing Status NAMETag" works too!
```

Comment:
I think it should be `spaces between the criteria` instead. Are multiple spaces ignored?